overrides/Breathing/ui/Breathing.ui

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "BreathingBar",
+  "skin": "HUD",
+  "contents": {
+    "type": "relativeLayout",
+    "contents": [
+      {
+        "type": "UIIconBar",
+        "id": "breathBar",
+        "icon": "CoreAssets:icons#bubble",
+        "family": "breathBar",
+        "emptyIcon": "CoreAssets:icons#burstBubble",
+        "halfIconMode": "shrink",
+        "spacing": 2,
+        "maxIcons": 10,
+        "layoutInfo": {
+          "use-content-width": true,
+          "use-content-height": true,
+          "position-horizontal-center": {},
+          "position-bottom": {
+            "target": "BOTTOM",
+            "offset": 102
+          }
+        }
+      }
+    ]
+  }
+}
```

Comment:
Trivial best practice: end all text type files with an empty newline :-)

Not something worth changing here, just FYI!

Reply:
> Trivial best practice: end all text type files with an empty newline :-)
> 
> Not something worth changing here, just FYI!

thank you! i will follow that from now on